proxy-node-gateway/src/main/java/uk/gov/ida/notification/GatewayApplication.java

```diff
@@ -120,6 +120,15 @@ private void setRequestServletFilter(Environment environment) {
     }
 
     private void setResponseServletFilter(Environment environment) {
+
```

Comment:
This filter fires last after all other filter chain processing if registered first in servlet environment.